rpmlb/cli.py

```diff
@@ -126,3 +135,15 @@ def run(recipe_file, collection_id, **option_dict):
     builder.run(work, **option_dict)
 
     log.info('Success!')
+
+
+def _adjust_option_dict(option_dict):
+    if not option_dict:
+        return
+    download = option_dict.get('download', None)
+    # When specifying download type: fedpkg or rhpkg,
+    # users must want to use the command in Builder.
```

Comment:
I tend to be explicit with `None`, as for me it signifies "unknown or unset value" in opposite to "set value that happens to be falsey", (i.e. empty string). But this boils down to personal preference, so I'm fine with your proposed solution too.